_layouts/conference.html

```diff
@@ -0,0 +1,83 @@
+    </h2>
+  </div>
+
+  <div class="grid__item width-12-12 sec-title-red">
+    <h2>What you need to know</h2>
+    <dl class="c-when-and-where">
+      <dt>Date</dt>
+      <dd id="c-date" data-date="2024-03-06">Wednesday, March 6, 2024</dd>
+      <dt>Time</dt>
+      <dd>14:00 - 18:00 UTC</dd>
+      <dt>Location</dt>
+      <dd>
+        Virtually on YouTube Live: <a href="{{site.baseurl}}/conference" target="_blank">https://link-to-you-tube</a>
+      </dd>
+    </dl>
+  </div>
+
+  <div class="grid__item width-12-12 sec-title-yellow">
+    <h2>What you can expect</h2>
+    <p>The conference includes four sessions with topics for both beginners and advanced WildFly users. There will
+      be an opening and closing ceremony and a 15-minute break between the second and third session. We want all
+      sessions to be interactive, and you can ask questions at any time. Remember this is a conference for you -
+      the WildFly community!</p>
+
+    <div class="admonitionblock note" style="margin-bottom: 2rem">
+      <table>
+        <tbody>
+        <tr>
+          <td class="icon">
+            <div class="title">Note</div>
+          </td>
+          <td class="content">
+            <div class="paragraph">
+              <p>All times are in <strong>UTC</strong>. Click on the time to show the time in your time
+                zone.</p>
+            </div>
+          </td>
+        </tr>
+        </tbody>
+      </table>
+    </div>
+
+    {% for session in site.data.conference %}
+    {%- capture session_bg -%}
+    {%- if session.abstract -%}c-session-blue{%- else -%}c-session-gray{%- endif -%}
+    {%- endcapture -%}
+    <div class="c-session {{ session_bg }}">
+      <div class="c-time">
+        <span class="c-from">{{ session.from }}</span><span class="c-to">{{ session.to }}</span>
+      </div>
+      <div class="c-title">{{ session.title}}</div>
+      {% if session.abstract %}
+      <div class="c-abstract"><p>{{ session.abstract | join: "<br/>" }}</p></div>
+      {% endif %}
+      {% if session.speaker %}
+      {% assign speaker = site.data.authors[session.speaker] -%}
+      <div class="c-speaker-avatar">
+        <img class="avatar" height="60px" width="60px" src="{{site.baseurl}}/assets/img/authors/{{ session.speaker }}.jpg"
+             alt="{{ speaker.name }}" title="{{ speaker.name }}"/>
+      </div>
+      <div class="c-speaker-bio">{{ session.bio }}</div>
+      {% endif %}
+    </div>
+    {% endfor %}
+  </div>
+
+  <div class="grid__item width-12-12 sec-title-teal">
+    <h2>After the conference</h2>
+    <p>Please take some time to answer a few questions about the conference. What did you like, what can we do
+      better? It won't take long, and will help us make the next WildFly conference even better: <a
+              href="{{site.baseurl}}/conference" target="_blank">conference feedback</a></p>
```

Comment:
I think on the preview right now the link for conference feedback opens the conference page in a new tab as opposed to the feedback form.